extension/package.json

```diff
@@ -54,6 +54,22 @@
         "group": "navigation"
       }
     ],
+    "configuration": {
+      "type": "object",
+      "title": "XComponent Configuration",
+      "properties": {
+        "xcbuild": {
+					"type": "string",
+					"default": "",
```

Comment:
the default value should be xcbuild.exe (when xcbuild.exe is in the path env) no ?

Reply:
I think it is more safe to put the entire path. So, even if user forget to add xcbuild.exe in path env, the command will work. @maurelio1234

Reply:
I am not sure to understand how it should work.
If I define nothing in the settings, it should try to use xcbuild in the path environment ?

But check your code in the project builder: 
if (!fs.existsSync(xcbuildPath)) {
         vscode.window.showErrorMessage(`xcbuild.exe not found. Please specify xcbuild path in vscode settings`);
         return;
    }
It's not going to work...

Reply:
If you specify xcbuild.exe in the path environnement, the vscode settings is 100% useless. 
The code works only if you set xcbuild path in vscode settings

Reply:
ok why not we can try with this implementation. But rename the mono setting because it is a wrong name
In a more standard way, the settings is not useless. It's a way to override some parameters:
- if there's a setting -> then take the setting
- no setting defined -> then try to use the environment path
it is pretty standard